### PR TITLE
kie-issues#565: Add apache license headers

### LINKED ISSUE #565: replace copyrights headers/comments in `kogito-website`

### PR DESCRIPTION
**Pull request by jomarko**
Closes: https://github.com/apache/incubator-kie-issues/issues/565

- [x] I installed [Jekyll](https://jekyllrb.com/docs/) locally
- [x] I ran `bundle exec jekyll build` before sending this PR

or

- [ ] I ran `docker run --rm --volume="$PWD:/srv/jekyll" -it -p 4000:4000 jekyll/jekyll:3 jekyll serve`

### CODE REVIEW

File: _site/assets/css/main.css [hunk 1]
```diff
@@ -1 +0,0 @@
...wrapper{padding:4rem 6rem 6rem 6rem}}@media screen and (max-width: 480px){.homepage-hero-band .wrapper{padding:4rem 2rem 6rem 2rem}}.homepage-hero-band h1,.homepage-hero-band h2{color:#fff}@media screen and (min-width: 768px){.homepage-hero-band h1{font-size:6rem;line-height:6rem}}.homepage-hero-band h2{font-family:'Josefin Sans', sans-serif;text-transform:uppercase;font-size:1.2rem;line-height:1;max-width:42rem}.homepage-hero-band a.button-cta,.homepage-hero-band a.button-cta:hover,.homepage-hero-band a.button-cta:focus,.homepage-hero-band a.button-cta:visited{border-color:#fff;margin-left:0}.tabs{display:flex;flex-wrap:wrap;margin:2rem 0}.tabs label{order:1;display:block;padding:1rem 2rem;margin-right:0.2rem;cursor:pointer;background-color:#daedf5;font-weight:700;transition:background ease 0.2s}.tabs .tab{order:99;flex-grow:1;width:100%;display:none;padding:1rem;background-color:#f6f5f5}.tabs input[type="radio"]{display:none}.tabs input[type="radio"]:checked+label{background-color:#f6f5f5}.tabs input[type="radio"]:checked+label+.tab{display:block;overflow:scroll}@media (max-width: 45em){.tabs .tab,.tabs label{order:initial}.tabs label{width:100%;margin-right:0;margin-top:0.2rem}}
```
Comment (porcelli) [inline _site/assets/css/main.css]: is this deleted file on purpose?

**Review by porcelli — COMMENTED** [review]
have one question

File: _site/assets/css/main.css (hunk 1 above)
Comment (jomarko) [reply _site/assets/css/main.css]: not on purpose, trying to revert, thank you for spotting

**Review by LightGuard — CHANGES_REQUESTED** [review]
Nothing in `_site` should be committed. That's the build output directory.

**Comment by LightGuard** [comment]
FYI, I cannot build this using the reference commands. Tried locally with ruby 3.2.2 and docker on M1 MBP.

**Comment by porcelli** [comment]
@jomarko we need figure out how best address the concern of building this somehow before merge =/

**Comment by jomarko** [comment]
@LightGuard @porcelli I have updated the PR. Thank you for an explanation that `_site` is autogenerated. These question comes to my mind related to it:
- shouldn't we then remove `_site` from the tracked files of this github repository?
- is `_site` somehow automatically updated  and committed via https://github.com/apache/incubator-kie-kogito-website/blob/main/.github/workflows/check-url.yml ? I went trough the yaml file, I see there `jekyll build` but not sure I see there somehow we update `_site` that we store on github

For me `docker jekyll serve` was also not working locally. I was able to run jekyll as `bundle exec jekyll build` however I needed to execute first step from https://github.com/apache/incubator-kie-kogito-website/tree/main/_site#installing-the-theme

**Comment by LightGuard** [comment]
Which version of Ruby?

**Comment by LightGuard** [comment]
I uncommented line 11 added `ruby "2.6.3"` right above in the Gemfile. That solved the docker issue for me.

**Comment by LightGuard** [comment]
We should also remove the `Gemfile.lock` as well as `_site` from the repo, but that's a different PR.

**Comment by jomarko** [comment]
@LightGuard ok, thank you for help with this PR, so what is the plan?
- review and possibly merge this PR? it is prepared for review from my point of view
- remove `Gemfile.lock` and `_site` in a new PR?

**Comment by LightGuard** [comment]
Well, after doing some research, Gemfile.lock needs to stay. This helps us ensure repeatable builds since the dependencies are locked.

I have the `_site` removed in my PR. I'm good with this being merged.

**Review by LightGuard — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

**Pull request merged**